charts/kafka-operator/templates/operator-deployment-with-webhook.yaml

```diff
@@ -199,9 +199,24 @@ spec:
           {{- if (.Values.metricEndpoint).port }}
             - --metrics-addr=":{{ .Values.metricEndpoint.port }}"
           {{- end }}
+          {{- if .Values.healthProbe.port }}
+            - --health-probe-addr=:{{ .Values.healthProbe.port }}
+          {{- end }}
           image: "{{ .Values.operator.image.repository }}:{{ .Values.operator.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.operator.image.pullPolicy }}
           name: manager
+          livenessProbe:
+            httpGet:
+              port: health-probes
```

Comment:
It might be nice to indicate this refers to health AND readiness in some way but it's not a requirement from me.

Reply:
I have chosen this way based on some resource (other codes) in which readiness and liveness probes are referred as health checks. But I can understand your point.

Reply:
Finally I kept the name because the name must be at most 15 characters long.